src/Microsoft.Diagnostics.EventFlow.Outputs.ApplicationInsights/Microsoft.Diagnostics.EventFlow.Outputs.ApplicationInsights.csproj

```diff
@@ -25,7 +25,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.8.1" />
+    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.14.0" />
```

Comment:
note that our latest version is 2.15 released on September 15th.

Reply:
Thanks. Yes, I know. We tend to use slightly older version than the latest to avoid triggering a dependency package upgrade (in this case, ApplicationInsights-dotnet) if one decides to upgrade an EventFlow package. 

That said, if you think we should take a dependency on 2.15, because of bug fixes or otherwise, just let me know and I will change that.